layersvt/device_simulation.cpp

```diff
@@ -73,16 +75,54 @@ const uint32_t kLayerPropertiesCount = (sizeof(kLayerProperties) / sizeof(kLayer
 const VkExtensionProperties *kExtensionProperties = nullptr;
 const uint32_t kExtensionPropertiesCount = 0;
 
+#if !defined(__ANDROID__)
```

Comment:
If this is for the unused variable, note that I have a change in the works that will use it.
So if the unused var is causing trouble, we can leave the #if in there for the moment, and I'll remove it in my upcoming PR.